src/pages/MainPage/MainPage.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 
 import Button from "../../Components/Button";
-import style from "./style.module.scss";
+import style from  "./style.module.scss";
 
 const MainPage = () => (
   <div className={style.boxFor}>
     <div className={style.boxBtnHi}>
       <div className={style.message}>Вітаю!</div>
-      <Button style={style.greetingBtn} value={"Увійти"} to="/logIn" />
+      <Button styles={style.greetingBtn} value={"Увійти"} to="/logIn" onClick={undefined} />
```

Comment:
`onClick` it should be a function. Howewer here we have undefined. Remove it if u don't need that onClick event or add correct function